### DISCUSSION TITLE
Discount codes invalid

### DISCUSSION THREAD

**Discussion by dfunnell1** [liedekef/events-made-easy#121] [Q&A]
Hi All,

I might be missing the blinding obvious, but I'm trying to set up a discount code for memberships. I have set up the code as a percentage type and included it in a discounts group. I have included this discount group in the membership, and the #_DISCOUNT in the membership form.
When I test this using the coupon code, it tells me the discount code is invalid (it's really not). 
Any ideas would be greatly appreciated.

**Comment by liedekef** [comment]
Bug found and fixed in dev: https://github.com/liedekef/events-made-easy/commit/3966d44e284c3ce50d3d8d6d5b2d47db53e95849
In fact it was fixed some time ago for events, but I overlooked to fix it for memberships too.

**Reply by liedekef** [reply]
I released a new version, it includes the fix.

**Reply by dfunnell1** [reply]
Brilliant, just updated and thats solved it. Thank you!
